src/app/modules/investigation-report-di/components/investigation-report-di-add/investigation-report-di-add.component.ts

```diff
@@ -912,31 +916,46 @@ export class InvestigationReportDiComponent implements OnInit {
     }//end of if
   }//end of the method onComplaintTypeSelect
 
-  // start method of onNatureTypeSelect
-  public onNatureTypeSelect(args, natureOfCompId: string) {
-    let natureCmpName: string = args.target.options[args.target.selectedIndex].text;
-    console.log("  this.natureCmpName ============================= ", natureCmpName);
-    console.log("natureOfCompId", natureOfCompId);
-    if (natureOfCompId && natureCmpName) {//checking if nature of comp id and name is avalable or not
-      this.tempCompTypeJson.natureOfComplaintId = natureOfCompId;
-      this.tempCompTypeJson.natureOfComplaintDesc = natureCmpName;
-      this.addModalCheckBoolean = true;
+  // start method to select comp type from edit item modal
+  public onEditItemComplaintTypeSelect(compTypeDropdownListFromEditModal){

```

Comment:
Is this method capturing both add and edit?
Then how are you handling?

Reply:
No this method is only used in edit modal.